build/image/photon/Dockerfile

```diff
@@ -9,6 +9,9 @@ RUN go build -o manager cmd/main.go
 
 FROM photon
 
+RUN tdnf -y install shadow && \
+    useradd -s /bin/bash nsx-operator
+
 COPY --from=golang-build /source/manager /usr/local/bin/
 
-ENTRYPOINT [ "/usr/local/bin/manager"]
+ENTRYPOINT ["su", "nsx-ncp", "-m", "-c", "/usr/local/bin/manager", "-nsxconfig", "/etc/nsx-ujo/ncp.ini", "-health-probe-bind-address", ":8384"]
```

Comment:
`/etc/nsx-ujo/ncp.ini`: Sorry, this does not look good to me, that we add ncp config in nsx-operator repo, so looks like we still need command in NCP yaml definition.
@heypnus any comments?

Reply:
ok